tests/backup/backup_stork_test.go

```diff
@@ -0,0 +1,569 @@
+package tests
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/pborman/uuid"
+	api "github.com/portworx/px-backup-api/pkg/apis/v1"
+	"github.com/portworx/sched-ops/k8s/core"
+	"github.com/portworx/torpedo/drivers/backup"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/pkg/log"
+	. "github.com/portworx/torpedo/tests"
+	v1 "k8s.io/api/core/v1"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// This testcase verifies backup and restore with non existing and deleted custom stork admin namespaces
+var _ = Describe("{BackupandRestoreWithNonExistingAdminNamespace}", func() {
```

Comment:
nit: change this to `BackupAndRestoreWithNonExistingAdminNamespace`

Reply:
Done